src/profile.js

```diff
@@ -254,6 +267,42 @@ var Profile = CreateClass({
     },
     render: function () {
         var locale = this.props.locale;
+        //Generate Critical Buff Fields
+        var criticalBuffRender = [];
+        if (this.state.criticalBuff.length != this.state.criticalBuffCount) this.state.criticalBuff = this.state.criticalBuff.slice(0, this.state.criticalBuffCount);
+        for (var i = 0; i < this.state.criticalBuffCount; i++) {
+            if (this.state.criticalBuff[i] == undefined) this.state.criticalBuff[i] = {"value": 0.0, "attackRatio": 0.0};
```

Comment:
- [ ] this "if" statement does not make indent.
- [ ] change state in render() view ... should be in event count changed timing
- [ ] can be function?(or react component) which take parameter criticalBuff, personalCriticalBuff

Reply:
> [ ]  this "if" statement does not make indent.

it's a oneline-er.